articles/channel.html

```diff
@@ -488,7 +488,7 @@ <h3>Detailed Explanations For Channel Operations</h3>
 
 <u><b>Channel operation case A</b>:
 when a goroutine <code>Gr</code> tries to receive a value from a not-closed non-nil channel</u>,
-the goroutine <code>Gr</code> will acquire the lock accociated with the channel firstly,
+the goroutine <code>Gr</code> will acquire the lock associated with the channel firstly,

```

Comment:
Aha, there is another such typo at line 535. ;D

Reply:
Done :)